cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -74,16 +74,16 @@ These functions take multiple values as arguments, and calculate and return an a
 [options="header"]
 |===
 |Function                                       | Description
-|<<functions-avg,avg()>>                        | Returns the average of a numeric column.
-|<<functions-collect,collect()>>                | Returns a list containing all collected values.
-|<<functions-count,count()>>                    | Returns the number of rows.
-|<<functions-max,max()>>                        | Returns the highest value in a numeric column.
-|<<functions-min,min()>>                        | Returns the lowest value in a numeric column.
+|<<functions-avg,avg()>>                        | Returns the average of a set of numeric values.
+|<<functions-collect,collect()>>                | Returns a list containing the values returned by a given expression.
+|<<functions-count,count()>>                    | Returns the number of values or rows.
+|<<functions-max,max()>>                        | Returns the maximum value in a set of values.
+|<<functions-min,min()>>                        | Returns the minimum value in a set of values.
 |<<functions-percentilecont,percentileCont()>>  | Returns the percentile of a given value over a group using linear interpolation.
 |<<functions-percentiledisc,percentileDisc()>>  | Returns the nearest value to a given percentile over a group using a rounding method.
 |<<functions-stdev,stDev()>>                    | Returns the standard deviation for a given value over a group for a sample of a population.
 |<<functions-stdevp,stDevP()>>                  | Returns the standard deviation for a given value over a group for an entire population.
-|<<functions-sum,sum()>>                        | Returns the sum of a numeric column.
+|<<functions-sum,sum()>>                        | Returns the sum of a set of non-`null` numeric values.
```

Comment:
Surely 'non-null' is relevant to more than just this expression? It could be true of all aggregating expressions, right?